control.go

```diff
@@ -0,0 +1,58 @@
+package bn
+
+import (
+	"context"
+	"time"
+
+	"github.com/libsv/go-bn/models"
+)
+
+// ControlClient interfaces interaction with the control sub commands on a bitcoin node.
+type ControlClient interface {
+	ActiveZMQNotifications(ctx context.Context) ([]*models.ZMQNotification, error)
```

Comment:
do you think we should separate the zmq stuff into their own thing? @Tigh-Gherr

Reply:
Yeah that's the plan, there will be a `zmq` sub directory or something.

Any reference to zmq at the comment are actually bitcoin node rpc functions, so no actual zmq work has been done yet. That's this weeks agenda.